src/commands/epgsql_cmd_prepared_query2.erl

```diff
@@ -0,0 +1,144 @@
+%% @doc Almost the same as equery, but don't execute 'CLOSE'
+%%
+%% So, statement can be reused multiple times.
+%% ```
+%% > Bind
+%% < BindComplete
+%% > Execute
+%% < DataRow*
+%% < CommandComplete
+%% > Sync
+%% < ReadyForQuery
+%% '''
```

Comment:
This doc loocs like plain copy-paste from `prepared_query`. please, adjust it with what the code actually does